oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -207,6 +207,13 @@ Do the following to document your proposal:
    deprecation spans multiple repos, choose the primary/most relevant repo, or
    use the `public-engineering`_ project.
 
+   .. note::
+      While it is possible to create Issues with no template, it is strongly
+      encouraged that you go to `github.com/openedx/:repo/issues`, click "New
+      Issue", and choose the DEPR template so you don't miss any fields and
+      automation works properly. If you must create an Issue outside the
```

Comment:
```suggestion
      automation works properly. The template fields help us more quickly
      address deprecation issues and reduce the amount of back and forth needed
      to make progress on work. If you must create an Issue outside the
```

Maybe being more clear about the exact benefits the template buys will make it clear why it's valuable?